### PR TITLE
add release notes

### PR DESCRIPTION
**Pull request by simmdan**

### CODE REVIEW

File: prosepy/docs-ref-conceptual/relnotes.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+# Release Notes
+
+## 1.1.0 - 2018-10-24
+
+- New configuration mechanism for specifying Code Accelerator options.  See [Configuration](config.md) for more details.
+- Renamed `.data` to `.preview_data` on all builders.  This property now returns a fixed number of rows (5).
+- The `prose.codeaccelerator.jupyter` namespace has been removed and its functionality is now included in the main
+  `prose.codeaccelerator` namespace.  When Code Accelerator is used in an environment that supports improved output
+  rendering, that rendering is displayed automatically.
+- Now supports Jupyterlab as well as Jupyter notebooks.
```
Comment (barik) [inline prosepy/docs-ref-conceptual/relnotes.md]: Pedantically, JupyterLab.

File: prosepy/docs-ref-conceptual/relnotes.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+- Generated code for pandas now uses `df.apply` on every column instead of `df.transform`.
+- Fixed some cases where a stack-overflow could result from attempting to coerce the types in a data frame.
+
+### FindPatternsBuilder
+
+- New `FindPatternsBuilder.include_outlier_patterns` property which optionally directs the builder to include additional
+  (possibly low quality) patterns in the returned result.
+- Generated code now includes examples and stats in comments.
+- Regex names produced have been simplified to make them easier to read.
```
Comment (barik) [inline prosepy/docs-ref-conceptual/relnotes.md]: Awkward? Maybe "Produced regex names...."

File: prosepy/docs-ref-conceptual/relnotes.md (hunk 1 above)
Comment (simmdan) [reply prosepy/docs-ref-conceptual/relnotes.md]: Good catch.  Fixed.

File: prosepy/docs-ref-conceptual/relnotes.md (hunk 2 above)
Comment (simmdan) [reply prosepy/docs-ref-conceptual/relnotes.md]: Fixed.

**Pull request merged**